target/scripts/startup/setup.d/saslauthd.sh

```diff
@@ -1,29 +1,29 @@
 #!/bin/bash
 
-
 function _setup_saslauthd() {
   _log 'debug' 'Setting up SASLAUTHD'
 
-  if [[ ! -f /etc/saslauthd.conf ]]; then
+  # NOTE: It's unlikely this file would already exist,
+  # Unlike Dovecot/Postfix LDAP support, this file has no ENV replacement
+  # nor does it copy from the DMS config volume to this internal location.
+  if [[ ${ACCOUNT_PROVISIONER} == 'LDAP' ]] \
+  && [[ ! -f /etc/saslauthd.conf ]]; then
```

Comment:
Technically instead of `ACCOUNT_PROVISIONER=LDAP`, this should be for `SASLAUTHD_MECHANISMS=ldap`, as the two features serve a separate purpose? 🤷‍♂️ (_although I'm not sure when you'd only have Postfix auth against LDAP while using a different `ACCOUNT_PROVISIONER`_).

Reply:
Should we take action here immediately?

Reply:
Nah, this is getting reworked by a follow-up and can be addressed by that 👍